src/adapters/southaustralia.js

```diff
@@ -0,0 +1,233 @@
+        console.log('Something went wrong, failed to resolve feed.');
+    }
+
+    const sites = new Set();
+  
+    feed.items.forEach((item) => {
+      const root = parse(item.content);
+      const head = root.querySelector('thead'); 
+      const keys = head.childNodes[0].childNodes.map((o) => o.text); 
+      const tbody = root.querySelector('tbody');
+      const rows = tbody.childNodes;
+      for (const row of rows) {
+        const cells = row.childNodes;
+        const site = Object.assign(
+          ...keys.map((k, i) => ({ [k]: cells[i].text }))
+        );
+        sites.add(site);
+      }
+    });
+  
+    let uniqueSites = [...sites];
+    uniqueSites.forEach((site) => {
+      site.time = DateTime.fromFormat(site['Date/time'], 'yyyy-MM-dd HH:mm:ss', { zone: 'Australia/Adelaide' });
+    });
+  
+    uniqueSites = getLatestEntries(uniqueSites);
+  
+    return uniqueSites;
+}
+  
+async function getLatestEntries(sites) {
+    const latestEntries = sites.reduce((result, site) => {
+        const key = site.Site;
+        const currentLatest = result[key];
+
+        if (!currentLatest || site.time > currentLatest.time) {
+        result[key] = site;
+        }
+
+        return result;
+    }, {});
+
+    return Object.values(latestEntries);
+}
+
+async function formatData(data) {
+    try {
+        data.forEach(site => {
+            const location = locations.find(location => location.label === site.Site);
+            if (location) {
+                site.lat = location.lat;
+                site.lng = location.lng;
+            }
+            Object.keys(site).forEach(key => {
+                const newKey = correctParam(key);
+                if (newKey !== key) {
+                  site[newKey] = site[key];
+                  delete site[key];
+                }
+              });
+        });
+        
+        let filteredData = [];
+
+        data.forEach(obj => {
+        acceptableParameters.forEach(param => {
+            filteredData.push({ 
+                location: obj.Site,
+                city: obj.Region,
+
+                parameter: param,
+                value: obj.hasOwnProperty(param) ? parseFloat(obj[param]) : null,
+                unit: param === ('pm10' || 'pm25') ? 'µg/m³': 'ppm',
+                date: {
+                    utc: obj.time.toUTC().toISO({suppressMilliseconds: true}),
+                    local: obj.time.toISO({suppressMilliseconds: true})
+                },
+                coordinates: {
+                    latitude: parseFloat(obj.lat),
+                    longitude: parseFloat(obj.lng)
+                },
+                attribution: [
+                    {
+                    name: "South Australia Environmental Protection Authority (EPA)",
+                    url: "https://data.sa.gov.au/data/dataset/recent-air-quality" 
+                    }
+                ],
+                averagingPeriod: {
+                    unit: "hours",
+                    value: 1
```

Comment:
This doesnt line up for CO, since its an 8 hour averaging period

Reply:
each parameter is stored separately so it is possible. You'll likely need to make `value` a dynamic value based on some sort of `if..else...` logic and toggle between 8 and 1.